examples/tagging/tagging_trainer.py

```diff
@@ -130,6 +131,15 @@ def train(self):
 
         while epoch < self.config_data.num_epochs:
             epoch += 1
+
+            try:
+                from texar.torch.data import Batch
```

Comment:
I think this is better imported outside the `while` loop. You can actually install it under the class scope.